app/models/dossier.rb

```diff
@@ -758,6 +751,10 @@ def author_is_administration(author)
     author.is_a?(Instructeur) || author.is_a?(Administrateur) || author.is_a?(SuperAdmin)
   end
 
+  def visible_by_administration?
+    (!hidden_by_administration? && (!hidden_by_user? && en_construction?)) || (!hidden_by_administration? && en_instruction?) || (!hidden_by_administration? && termine?)
```

Comment:
Ouch, t'as pas fait simple sur ce coup là, j'espère que t'as un test car je vais tenter de simplifier

```suggestion
  !(hidden_by_administration? || en_brouillon?)
```

Reply:
Je pense que t'a oublié des cas. Ma proposition :

`!(hidden_by_administration? || (hidden_by_user? && en_construction?))`